content/posts/Vivek_Kumar_GSoC2025_FinalReport.md

```
@@ -0,0 +1,190 @@
+
+Hello Everyone😊,
+
+Welcome to my final blog for Google Summer of Code 2025 for the project **Enhanced Verilog Support & Stability**
+
+### What is the Project about🖥
+---
+
+> **[_Enhanced Verilog Support & Stability:_](https://summerofcode.withgoogle.com/programs/2025/projects/9cZjeC5m)
+> The project works on the Verilog feature of CircuitVerse namely Circuit to Verilog and Verilog to Ciruit and its stability. The project works on taking the verilog feature to production, adding verilog modules for the circuit elements and removing bugs in the feature. The project also introduces new features like Verilog terminal, play/pause button to the simulator, improves the UI/UX of the verilog code-editor, updates the yosys repo, fixes bug ,implements full screen view for tools window and documents the verilog feature.**
+
+### New features that have been added and merged include:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)